paypro-gateways-woocommerce/includes/paypro/wc/gateway/ideal.php

```diff
@@ -32,9 +26,9 @@ public function payment_fields()
         $selected_issuer = $this->getSelectedIssuer();
 
         $html = '<select name="' . PayPro_WC_Plugin::PLUGIN_ID . '_issuer_' . $this->id . '">';
-        $html .= '<option value=""></option>';
+        $html .= '<option value="" hidden></option>';
```

Comment:
I am not sure if I get this, could you make a screenshot for it? is it always hidden?

Reply:
it will still be selected as `default` bc its the first option in the list but you wont see the `empty` option in the list of options

Reply:
![image](https://user-images.githubusercontent.com/55430516/142639975-cd53b6c5-8784-4643-a4cf-ae20aaf9c44e.png)

Reply:
![image](https://user-images.githubusercontent.com/55430516/142640091-875cd59e-b83e-48ff-8d4b-c49b1817a685.png)